plugin.php

```diff
@@ -1,6 +1,12 @@
 <?php
 /**
- * Cavalcade!
+ * Plugin Name: Cavalcade
+ * Plugin URI: https://github.com/humanmade/Cavalcade
+ * Description: Scalable job system, designed as a drop-in replacement for wp_cron
+ * Version: 0.1
```

Comment:
I'll check if it works even without the version. I'll use `dev-master` if it's mandatory.